### DISCUSSION TITLE
Snippet Extension Deprecation

### DISCUSSION THREAD

**Discussion by danremollino** [zed-industries/zed#15888] [Feature Requests]
Zed release [0.144.3](https://zed.dev/releases/stable/0.144.3) has a release note mentioning...

- Move snippet support into the core editor deprecating the snippet extension. Snippets are now available in any buffer including plain text ([#13937](https://github.com/zed-industries/zed/pull/13937)).

so I should be able to remove the extension, correct? If I do so, I can't get any of my snippets to work at all.

**Comment by fuzenco** [comment]
I had the same question. So I removed my previously installed Snippets extension and the good news: snippets continue to work.

**Reply by danremollino** [reply]
> I had the same question. So I removed my previously installed Snippets extension and the good news: snippets continue to work.

Just gave another test. My snippets absolutely do not work without the extension installed. I'm using a snippets.toml file. Is toml still supported?

**Reply by fuzenco** [reply]
Don’t know. I’m using .json.

**Reply by danremollino** [reply]
Got around to doing some testing and looks like snippets.json files work and snippets.toml files no longer work when not using the extension.

@osiewicz is this intentional?

**Reply by osiewicz** [reply]
Yes, we removed support for .toml snippets.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
